lib/pages/chat/sticky_timestamp_widget.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/colors/linagora_ref_colors.dart';
+
+class StickyTimestampWidget extends StatelessWidget {
+  final String content;
+  final bool isStickyHeader;
+
+  const StickyTimestampWidget({
+    Key? key,
+    required this.content,
+    this.isStickyHeader = false,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+      alignment: Alignment.topCenter,
+      child: UnconstrainedBox(
+        child: Opacity(
+          opacity: isStickyHeader ? 0.8 : 1.0,
```

Comment:
create style file

Reply:
or put it in style

Reply:
IMO: It's a small widget, no need to create style file

Reply:
lol you just lazy